lbrynet/reflector/server/server.py

```diff
@@ -47,15 +113,15 @@ def _get_valid_response(self, response_msg):
         extra_data = None
         response = None
         curr_pos = 0
-        while True:
+        while not self.receiving_blob:
             next_close_paren = response_msg.find('}', curr_pos)
             if next_close_paren != -1:
                 curr_pos = next_close_paren + 1
                 try:
                     response = json.loads(response_msg[:curr_pos])
                 except ValueError:
-                    if curr_pos > 100:
-                        raise Exception("error decoding response")
+                    if curr_pos > 200:
```

Comment:
client queries can be more than 100 bytes,
{
     'sd_blob_hash': '48d568facff0400581a1b449b4c624da061d9783063e118764b556c27f99fad3855f74155cf0ff431d3486580e279e13', 
    'sd_blob_size': 2097152
}

is 141